src/chatgpt.ts

```diff
@@ -64,6 +69,9 @@ export class ChatGPTBot {
       });
     console.log(`Chatgpt pool size: ${chatGPTPools.length}`);
     this.chatGPTPools = chatGPTPools;
+    config.botConfig.forEach((item: { [key: string]: string; }) => {
```

Comment:
`this.botConfig = {...config.botConfig,...this.botConfig}`?

Reply:
modify to 
`
`

Reply:
`
   config.botConfig.forEach((item: { [key: string]: string; }) => {
        Object.keys(item).forEach((configItemKey: string ) =>{
            this.botConfig[configItemKey] = item[configItemKey]
        })
    });
`

Reply:
Try use `Object.assign({}, config.botConfig, this.botConfig)` pls.

Reply:
The problem has been fixed.